src/wp-includes/js/media/controllers/customize-image-cropper.js

```diff
@@ -4,14 +4,20 @@ var Controller = wp.media.controller,
 /**
  * wp.media.controller.CustomizeImageCropper
  *
- * @memberOf wp.media.controller
+ * @summary Prepares the image to get cropped.
+ *
+ * Checks whether the image needs to be cropped according to a ratio.
  *
- * A state for cropping an image.
+ * @since 4.3
+ *
+ * @memberOf wp.media.controller
  *
  * @class
  * @augments wp.media.controller.Cropper
  * @augments wp.media.controller.State
  * @augments Backbone.Model
+ *
+ * @returns ajax post Call to Ajax to crop the image according to the crop details.
```

Comment:
I guess this should a `jQuery.promise`.  See : https://github.com/Yoast/wordpress-develop-mirror/pull/82#discussion_r147137062